src/sdk/sdk.interface.ts

```diff
@@ -63,6 +64,7 @@ export interface DialectCloudConfigProps {
   environment?: DialectCloudEnvironment;
   url?: string;
   tokenStore?: TokenStoreType | TokenStore;
+  tokenLifetime?: Duration;
```

Comment:
I don't think it's a good idea to use luxon in public api, since it'll force everyone who wants to configure this parameter to explicitly install luxon and use it

Reply:
Oh, that's a great point!

Pass duration in millis and convert to Duration? Or even so, simply use basic math to perform the comparison?

Reply:
Nice point, updated